src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import PlainButton from '../../../../components/plain-button';
+import ArrowArcRight from '../../../../icon/fill/ArrowArcRight';
+
+import type { GetProfileUrlCallback } from '../../../common/flowTypes';
+import type { SelectorItems, User } from '../../../../common/types/core';
+
+import CommentForm from '../comment-form';
+
+import messages from './messages';
+import './ActivityThreadReplyForm.scss';
+
+type Props = {
+    currentUser?: User,
+    getMentionWithQuery?: Function,
+    getUserProfileUrl?: GetProfileUrlCallback,
+    mentionSelectorContacts?: SelectorItems<>,
+    onReplyCreate: Function,
+};
+
+export default function ActivityThreadReplyForm({
+    currentUser,
+    mentionSelectorContacts,
+    getMentionWithQuery,
+    onReplyCreate,
+}: Props) {
+    const [showReplyForm, setShowReplyForm] = React.useState(false);
+    const placeholder = useIntl().formatMessage(messages.replyInThread);
```

Comment:
We don't currently support `useIntl` for backward-compatibility reasons. Can we switch to `injectIntl`?